airgun/widgets.py

```diff
@@ -727,8 +727,9 @@ class SatFlashMessages(FlashMessages):
 
     """
 
-    ROOT = '//div[@class="toast-notifications-list-pf"]'
-    MSG_LOCATOR = f'{ROOT}/div[contains(@class, "alert")]'
+    ROOT = '//div[@class="pf-c-alert pf-m-success foreman-toast" or \
+        data-ouia-component-type="PF4/Alert"]'
```

Comment:
A Better locator for `//ul[@class="pf-c-alert-group pf-m-toast"]/li` this will provide all list of alerts. You are making ROOT as one node level below and you are making a success alert, It could be an error alert as well.

Reply:
OK, will try that

Reply:
This change works:

     +    ROOT = '//ul[@class="pf-c-alert-group pf-m-toast"]'
     +    MSG_LOCATOR = f'{ROOT}/div[contains(@class, "pf-c-alert")]'

but I want to try with `data-ouia-component-type="PF4/Alert"`, I think I need @ in front.

Reply:
I could not get those pf4 locators to work, but will paste test results as new comment after above change.

Reply:
Concerning the PF4 locators, you would need to update the class inheritance. It doesn't look like a PF4 flash messages widget has been written for wt.pf4 OUIA module. 

https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/ouia.py

We would start with wrapping FlashMessages + OUIAGenericWidget into a `FlashMessages` class that defines OUIA_COMPONENT_TYPE.

@latran I would set this as part of the PF4 migration for airgun with a card in JIRA, and not force that update in this PR if @swadeley changes to the locator here are sufficient for passing test cases.